flytekit/common/tasks/sagemaker/training_job_task.py

```diff
@@ -0,0 +1,199 @@
+from __future__ import absolute_import
+
+from typing import Dict, Callable
+import datetime as _datetime
+
+from flytekit import __version__
+from flytekit.common.tasks import task as _sdk_task, sdk_runnable as _sdk_runnable
+from flytekit.models import task as _task_models
+from flytekit.models import interface as _interface_model
+from flytekit.common import interface as _interface
+from flytekit.models.sagemaker import training_job as _training_job_models
+from google.protobuf.json_format import MessageToDict
+from flytekit.models import types as _idl_types
+from flytekit.models.core import types as _core_types
+from flytekit.models import literals as _literal_models
+from flytekit.common.constants import SdkTaskType
+
+
+class SdkSimpleTrainingJobTask(_sdk_task.SdkTask):
```

Comment:
What does Simple mean?

Reply:
is this meant to be built in?

Reply:
Yes -- meaning users don't write his/her own decorated function.